[2023-02-04 09:15:01,451] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-new-dag.bash_task scheduled__2023-02-04T09:10:00+00:00 [queued]>
[2023-02-04 09:15:01,462] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-new-dag.bash_task scheduled__2023-02-04T09:10:00+00:00 [queued]>
[2023-02-04 09:15:01,463] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 09:15:01,463] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-02-04 09:15:01,464] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 09:15:01,487] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): bash_task> on 2023-02-04 09:10:00+00:00
[2023-02-04 09:15:01,492] {standard_task_runner.py:52} INFO - Started process 11092 to run task
[2023-02-04 09:15:01,497] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-new-dag', 'bash_task', 'scheduled__2023-02-04T09:10:00+00:00', '--job-id', '263', '--raw', '--subdir', 'DAGS_FOLDER/new_dag.py', '--cfg-path', '/tmp/tmp9f356ubi', '--error-file', '/tmp/tmpy7opz8fh']
[2023-02-04 09:15:01,498] {standard_task_runner.py:80} INFO - Job 263: Subtask bash_task
[2023-02-04 09:15:01,585] {task_command.py:371} INFO - Running <TaskInstance: yandex-new-dag.bash_task scheduled__2023-02-04T09:10:00+00:00 [running]> on host 39df186e1726
[2023-02-04 09:15:01,682] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-new-dag
AIRFLOW_CTX_TASK_ID=bash_task
AIRFLOW_CTX_EXECUTION_DATE=2023-02-04T09:10:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-04T09:10:00+00:00
[2023-02-04 09:15:01,685] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-02-04 09:15:01,687] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'Hello, World!'"]
[2023-02-04 09:15:01,698] {subprocess.py:85} INFO - Output:
[2023-02-04 09:15:01,700] {subprocess.py:92} INFO - Hello, World!
[2023-02-04 09:15:01,701] {subprocess.py:96} INFO - Command exited with return code 0
[2023-02-04 09:15:01,739] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-new-dag, task_id=bash_task, execution_date=20230204T091000, start_date=20230204T091501, end_date=20230204T091501
[2023-02-04 09:15:01,789] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-04 09:15:01,824] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
